### ISSUE TITLE
Clarify salary methodology

### ISSUE THREAD

**Issue by mguthaus** [CSStipendRankings/CSStipendRankings#43]
I believe there needs to be some clarification and enforcement of methods to compute the data. For example, all University of California schools have the same contract for salaries so the minimum will likely be the same unless all students are guaranteed some outside fellowship in addition to salary.

1) Are institutions required to provide the minimum support or typical? For example, often TA positions get paid less than a GSR. If any student is supported solely by TA, is that the minimum bound and is required to be presented?

2) Are institutions allowed to include summer salary if summers are not guaranteed?

3) There should be a requirement to provide a link to verify numbers for salary.

**Comment by polochau** [comment]
Agreed with @mguthaus . Methodology clarifications similarly needed for "Living Cost" values too, to avoid doubling counting an expense in both "Living Cost" and "Fees" column, which is [happening for health insurance for at least for CMU and GT](https://github.com/CSStipendRankings/CSStipendRankings/issues/35#issuecomment-1537192786).

**Comment by noah-curran** [comment]
> Methodology clarifications similarly needed for "Living Cost" values too, to avoid doubling counting an expense in both "Living Cost" and "Fees" column

https://github.com/CSStipendRankings/CSStipendRankings/issues/17#issuecomment-1537175901 has some additional insights to this issue as well. There is definitely a need to provide some sort of academic adjustment to the MIT CoL calculator.

**Comment by mjc0608** [comment]
@mguthaus, our current plan is to collect data for the amount of summer funding, and whether a department guarantees summer funding. Then, we will implement a UI that allows a user to select one of the following as the stipend: (1) the money that a typical student will get annually, (2) an 8/9 month stipend without summer funding, and (3) only the guaranteed money. I actually don't know how to consider the stipend difference between TA and RA, since different universities have different rules and there are too much data to collect and organize.

@polochau, as I mentioned before, the MIT living cost model is something that's fair for all universities and the only such metric we are able to find for now. Including the cost of health insurance in fees is an intentional decision, and it is because most universities provide it at the university's cost. In the future, we will add other metrics (e.g., one based on the price of typical 2b2b apartments and the price of General Tso's Chicken), and users should be able to select which metric to use in the website UI.

**Comment by shibo-chen** [comment]
@polochau I agree with what @mjc0608 elaborated above. Since most of the universities have provided health insurance as part of the benefits and are not counted towards the total stipend, it's only fair to count the insurance fee as a mandatory fee to level the ground.

**Comment by polochau** [comment]
Thanks @mjc0608 and @shibo-chen . I'm sorry to have to re-iterate that it is wrong to double count health insurance, once in Fees and another time in Living Cost. It is possible to request the data of Living Wage Calculator https://livingwage.mit.edu/pages/faqs . I hope doing so will help shed light on exactly what medical related expenses are included in the outputs of the calculator.

**Comment by mjc0608** [comment]
It’s not fair to simply remove the health care cost from living cost,
because most insurances do have copays and most people still need dental
care, etc. it’s hard to say how much we should cut.

**Comment by shibo-chen** [comment]
@polochau After looking into MIT Living Wage's documentation https://livingwage.mit.edu/resources/Living-Wage-Users-Guide-Technical-Documentation-2023-02-01.pdf, they actually assumed the employer sponsors/subsidizes health insurance. Since CMU and Gatech do not include health insurance as benefits, we are actually right to count them as extra fees, so we are not double counting.

**Comment by polochau** [comment]
Thanks @shibo-chen . Page 5 of the pdf you provided says "The health component of the basic needs budget includes: (1) health insurance costs for employer sponsored plans, (3) medical services, (3) drugs, and (4) medical supplies." This does not mean the employer pays for all of the health insurance premium, because the subsidy may cover only part (and not all) of the premium; in other words, a employee may still need to pay some insurance premium (the best way to find out is to request the data behind the calculator and we will know exactly what numbers the calculator is using). 

It is wrong to say CMU and GT (or some other universities) do not offer health insurance as benefits --- while they do not fully pay for the whole insurance premium, they offer subsidized health plans (I wish they would fully cover all premium). 

In summary, again, health insurance is appearing in Fees, and then appear again in Living Cost. (The two columns are not using the same amount as you already know, since Living Cost is using an estimate that we could find out via data request, while Fee is using exact amount).

@mjc0608 I agree it's not easy, but only by digging into the detail would we know exactly what numbers the Living Cost calculator is using.

**Comment by mjc0608** [comment]
>It is wrong to say CMU and GT (or some other universities) do not offer health insurance as benefits --- while they do not fully pay for the whole insurance premium, they offer subsidized health plans (I wish they would fully cover all premium).

Agreed. We will considering move health insurance out of the fee and have a new column of health insurance costs. As I mentioned before, we need to be fair for all universities.

>only by digging into the detail would we know exactly what numbers the Living Cost calculator is using.

This won't happen in the near future unless contributed by a volunteer. And everyone is welcome to contribute.

**Comment by polochau** [comment]
Thanks @mjc0608 ! I support presenting accurate, fair information for all universities. I understand the challenge, and I appreciate all volunteers' effort.

**Comment by pyjhzwh** [comment]
> > It is wrong to say CMU and GT (or some other universities) do not offer health insurance as benefits --- while they do not fully pay for the whole insurance premium, they offer subsidized health plans (I wish they would fully cover all premium).
> 
> Agreed. We will considering move health insurance out of the fee and have a new column for health insurance costs. As I mentioned before, we need to be fair for all universities.
> 
> > only by digging into the detail would we know exactly what numbers the Living Cost calculator is using.
> 
> This won't happen in the near future unless contributed by a volunteer. And everyone is welcome to contribute.

I think it's also worth the add notes or a link to the health insurance provided by each university (the basic health insurance provided, and the premium upgrades). We could have a metric to calculate how much additional fees the students need to pay so that the coverage is similar across different universities. The use that number as the "health insurance" cost. What do you think? @mjc0608 @jiong-zhu @polochau @shibo-chen @TonyZhangND

**Comment by polochau** [comment]
I support clarifications like these!

**Comment by noah-curran** [comment]
> We could have a metric to calculate how much additional fees the students need to pay so that the coverage is similar across different universities.

After re-reading what was mentioned here a few times, this seems like the best option proposed. I suspect gathering the necessary data to achieve this will be substantial, so I wouldn't expect this result soon, but it's something we should work towards in the coming weeks.

**Comment by SolalPirelli** [comment]
Given that the goal of the website is to incentivize universities to increase pay, perhaps the methodology should be the _minimum guaranteed salary_? That is, regardless of what options might be available for funding in general, only display what a student is 100% sure to get regardless of who is advising them, how well they negotiate, what opportunities arise, etc.

**Comment by noah-curran** [comment]
> Given that the goal of the website is to incentivize universities to increase pay, perhaps the methodology should be the minimum guaranteed salary?

I like the idea. I think there is room to complain from universities who pay most students above the so-called guaranteed minimum, but perhaps it would incentivize them to bring everyone up to a higher amount if they're most of the way there.

**Comment by TonyZhangND** [comment]
@SolalPirelli @noah-curran Are there examples about schools who don't pay everyone the same amount?

**Comment by mguthaus** [comment]
@TonyZhangND At any UC, the salary tables for TA and GSR never align so the salary will depend on how you are funded.

**Comment by TonyZhangND** [comment]
Gosh that sucks :/ indeed in such case we should take the minimum guaranteed amount then.

**Comment by noah-curran** [comment]
> @SolalPirelli @noah-curran Are there examples about schools who don't pay everyone the same amount?

When I was admitted to Purdue ECE in 2020, I received an offer closer to ~$36k. (Can't remember the exact figure and don't believe I still have the offer on-hand.) It was noted that it was a "special fellowship", so it was outside of their usual offer. I think at the time this was around $25k (and now is $26k according to the data collected here). This is a personal anecdote that immediately stands out to me.

**Comment by TonyZhangND** [comment]
I see. Reminds me of another situation --- when I got an offer from Wisconsin Madison, it came with a scholarship that paid an extra 3k in the first year. Bonuses of this nature that doesn't last through the entire PhD also shouldn't count.

**Comment by SolalPirelli** [comment]
> @SolalPirelli @noah-curran Are there examples about schools who don't pay everyone the same amount?

ETH Zurich: https://ethz.ch/en/the-eth-zurich/working-teaching-and-research/welcome-center/employment-contract-and-salary/salary.html

I'm not 100% sure of what the rates mean but I believe it differs by subject and by advisor and by what one can negotiate.

**Comment by mjc0608** [comment]
I think the salary methodology is kind of finalized now. Also, we have the functionality to only display the guaranteed minimum. I'm closing this issue.
